src/Subscriber/FilterShopwareExceptionIgnoresSubscriber.php

```
@@ -0,0 +1,43 @@
+<?php
+
+namespace Frosh\SentryBundle\Subscriber;
+
+use Frosh\SentryBundle\Integration\FilterExceptionEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class FilterShopwareExceptionIgnoresSubscriber implements EventSubscriberInterface
+{
+
+    public function __construct(
```

PHPStan (2.1.15)

Method Frosh\SentryBundle\Subscriber\FilterShopwareExceptionIgnoresSubscriber::__construct() has parameter $shopwareExceptionConfig with no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)